include/deal.II/grid/grid_tools.h

```diff
@@ -661,6 +661,58 @@ namespace GridTools
                               const typename Triangulation<dim, spacedim>::active_cell_iterator &cell_hint
                               = typename Triangulation<dim, spacedim>::active_cell_iterator());
 
+  /**
+   * Distributed compute point locations is a function similar to
+   * GridTools::compute_point_locations but working for parallel::Triangulation
+   * objects and, unlike its serial version, also for a distributed triangulation
+   * ( see parallel::distributed::Triangulation ).
+   * Given a @p cache and a list of
+   * @p local_points for each process, find the points lying on the locally owned
+   * part of the mesh and compute the quadrature rules for them.
+   *
+   * @param[in] cache a GridTools::Cache object
+   * @param[in] local_points the array of points owned by the current process. Every
+   *  process can have a different array of points which can be empty and not
+   *  contained within the locally owned part of the triangulation
+   * @param[in] the description of the locally owned part of the mesh made with bounding
+   *  boxes. It can be obtained from GridTools::compute_mesh_predicate_bounding_box
+   * @param[out] tuple containing the quadrature information
+   *
+   * The elements of the output tuple are:
+   * - cells : a vector of a vector cells of the all cells containing at
```

Comment:
a vector of ~~a vector cells of the all~~ cells containing at least a single point. (?)